yarn-project/simulator/src/mocks/fixtures.ts

```diff
@@ -14,18 +14,18 @@ import { makeAztecAddress, makeSelector } from '@aztec/circuits.js/testing';
 import { FunctionType } from '@aztec/foundation/abi';
 import { padArrayEnd } from '@aztec/foundation/collection';
 
-import { type PublicExecution, type PublicExecutionResult } from '../public/execution.js';
+import { type PublicExecutionRequest, type PublicExecutionResult } from '../public/execution.js';
 
 export class PublicExecutionResultBuilder {
-  private _execution: PublicExecution;
+  private _execution: PublicExecutionRequest;
```

Comment:
Maybe for later: rename `execution` to request? or is it something different? I think it had some form or call request inside